front/src/modules/ui/tooltip/__stories__/Tooltip.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { userEvent, within } from '@storybook/testing-library';
+
+import { CatalogDecorator } from '~/testing/decorators/CatalogDecorator';
+import { ComponentDecorator } from '~/testing/decorators/ComponentDecorator';
+
+import { AppTooltip, TooltipValues } from '../AppTooltip';
+
+const meta: Meta<typeof AppTooltip> = {
+  title: 'UI/Tooltip/Tooltip',
+  component: AppTooltip,
+};
+
+export default meta;
+type Story = StoryObj<typeof AppTooltip>;
+
+export const Default: Story = {
+  args: {
+    place: 'bottom',
```

Comment:
position: {TooltipPosition.Bottom}

Reply:
Hey @charlesBochet ,
Did you mean we should use the TooltipPosition enum here instead of using 'bottom' directly? (Cause the position prop expects x and y values specifying the position of the tooltip)

Reply:
yes, sorry: I didn't know that it was called "place" in react-tooltip. Let's call everything TooltipPlace to be consistent